packages/cli/src/command.js

```diff
@@ -43,6 +49,19 @@ export default class NuxtCommand {
     }
 
     return Promise.resolve(this.cmd.run(this))
+      .then(() => {
+        if (this.forceExit) {
+          const exitTimeout = setTimeout(() => {
+            let msg = `The command 'nuxt ${this.cmd.name}' finished but Nuxt.js did not exit after ${forceExitTimeout}s\n`
+            msg += 'This is most likely not caused by a bug in Nuxt\n'
+            msg += 'Make sure to cleanup all timers and listeners you or your plugins/modules start.\n'
+            msg += 'Nuxt.js will now force exit'
+            process.stderr.write(warningBox(msg))
+            process.exit(0)
```

Comment:
Is it a good idea to silently exit with code 0?

Reply:
I did this on purpose because of the complaints in the issue I mentioned above, exiting with non-zero exit code would still be a breaking change for them compared to previous nuxt behaviour. I would propose to leave it at 0 for now and change it to non-zero in v3.0.

Reply:
@pimlie Seems reasonable. Let's do this for v3.0. Feel free adding a `TODO` statement above.